test/modules/Math/signbit-test.good

```diff
@@ -0,0 +1,8 @@
+1
+0
+1
+0
+1
+0
+0
+0
```

Comment:
This test fails for me. I see the output:

```
8
0
128
0
128
0
0
0
```

Reply:
@mppf  @ronawho 
Is it because of the fact that 
> The C version returns a Nonzero integral value if arg is negative, ​0​ otherwise. The difference being that the C version can return any non-0 value if x is negative

as you talked earlier about? I will force the function to convert non-0 values to 1 . However, should I do the task in `runtime/include/chplmath.h` or in `modules/standard/Math` ? 

I am also having doubts about if I define a macro in the `.h` files in `runtime/include` ,  which version of the function called will be run ? How to call the `signbit` of C++ and not of C ?

As far as I can understand, the `extern` keyword is for C interoperability, so C++ functions or macros won't be called ?

Reply:
chplmath.h includes math.h, so you'll always be getting the C version. I only mentioned the C++ version above because it has a different API that we may want to follow (we may want to return a bool instead of arbitrarily ranged int.)

Reply:
Ok, instead of this, 

> I will force the function to convert non-0 values to 1 . However, should I do the task in `runtime/include/chplmath.h` or in `modules/standard/Math` ?

should I do 

> (we may want to return a bool instead of arbitrarily ranged int.)
 
?  I am fine by both the approaches, so I need some more opinion on which one would others prefer more. You said you would lean towards the c++ interface, so just confirm it, and I will make the changes.

Reply:
I don't have a strong opinion here (or great intuition for API design.) Choosing the right interface is part of the challenge so feel free to make a proposal for which one you think is right and then ask for feedback.

Reply:
@ronawho Please take a look. I have updated the code.